### ISSUE TITLE
Not able to join server due to (what it seems) packet size exceeding limit

### ISSUE THREAD

**Issue by Tecohg** [DistantDig/digs-dnd-origins#27]
I am using origins, origins++ and your origins mod, though when i add the dnd origins, i can't join anymore, i would've used xl packet expander but the mod is not up to date, heres the error code if you need to:

java.lang.IllegalArgumentException: Payload may not be larger than 1048576 bytes
	at net.minecraft.class_2658.<init>(class_2658.java:39) ~[server-intermediary.jar:?]
	at net.fabricmc.fabric.impl.networking.server.ServerNetworkingImpl.createPlayC2SPacket(ServerNetworkingImpl.java:45) ~[fabric-networking-api-v1-1.3.8+b3afc78b77-9a2136a8d11229d.jar:?]
	at net.fabricmc.fabric.api.networking.v1.ServerPlayNetworking.createS2CPacket(ServerPlayNetworking.java:388) ~[fabric-networking-api-v1-1.3.8+b3afc78b77-9a2136a8d11229d.jar:?]
	at net.fabricmc.fabric.api.networking.v1.ServerPlayNetworking.send(ServerPlayNetworking.java:427) ~[fabric-networking-api-v1-1.3.8+b3afc78b77-9a2136a8d11229d.jar:?]
	at net.minecraft.class_3324.handler$zbj000$apoli$syncPowerTypes(class_3324.java:1051) ~[server-intermediary.jar:?]
	at net.minecraft.class_3324.method_14570(class_3324.java:307) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_33800(class_3248.java:131) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_14384(class_3248.java:118) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.redirect$zgm000$fabric-networking-api-v1$handlePlayerJoin(class_3248.java:563) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_18784(class_3248.java:68) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_10754(class_2535.java:259) ~[server-intermediary.jar:?]
	at net.minecraft.class_3242.method_14357(class_3242.java:172) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:908) ~[server-intermediary.jar:?]
	at net.minecraft.class_3176.method_3813(class_3176.java:283) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
	
	
	I hope you have solutions, either way thanks for fixing the dragon breath crash ! :D

**Comment by DistantDig** [comment]
im sorry youre having so many issues... This seems to me like origins++ pushes the limits on the packet size, and when you install dnd origins it pushes it over the edge. If you could do me a favor and try to run the server without origins++ but with dnd origins installed, that should tell us if im right

**Comment by DistantDig** [comment]
I've looked through origins++ github and this sort of thing seems to be a very common issue, since several people would try to run it without xl packet. If the server works with just origins, then it would seem that you just need a mod to replace xl packets. I've found [connectivity](https://www.curseforge.com/minecraft/mc-mods/connectivity) which increases the packet size among other things, but I'm not sure by how much. I'd say give it a try next to see if it fixes it
